apps/web/src/lang/fr.json

```diff
@@ -267,6 +267,10 @@
     "defaultMessage": "Retirer la compétence {name} de l’expérience",
     "description": "Aria-label for remove skill button from experience"
   },
+  "/Mp09M": {
+    "defaultMessage": "<hidden>Processes de recrutement pour</hidden> {poolName}",
```

Comment:
Space should be in hidden text: `<hidden>Processes de recrutement pour </hidden>{poolName}`

Reply:
Nice catch! Fixed in [cd98a19](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/cd98a19c3e3ee182c0b81d16570dc25c194c723a)